dasbus/typing.py

```diff
@@ -344,5 +360,6 @@ def _check_if_valid_dictionary(type_hint):
 
         if DBusType._is_container_type(key) or key == Variant:
             raise TypeError(
-                "Dictionary key cannot be of type {}.".format(key)
+                "Invalid DBus type '{}' of dictionary "
+                "key.".format(get_type_name(key))
```

Comment:
Looking at the strings in tests, the type might get really long, and move the "of dictionary" far away from the rest of the message. How about `"Invalid... key: {}"` ?

Reply:
Fixed.